library/options.inc.php

```diff
@@ -3966,4 +3966,28 @@ function lbf_canvas_head($small = true)
     return $s;
 }
 
+/**
+ * Test if modifier($test) is in array of options for data type.
+ *
+ * @param json array $options or could be string of form "ABCU"
+ * @param string $test
+ * @return boolean
+ */
+function isOption($options, $test)
+{
+    if (empty($options) || empty($test)) {
+        return false; // why bother?
+    }
+    if (strpos($options, ',') === false) { // could be string of char's or single element of json
+        json_decode($options);
+        if (is_string($options) && ! (json_last_error() === JSON_ERROR_NONE)) { // nope, it's string.
+            $t = str_split(trim($options)); // very good chance it's legacy modifier string.
+            $options = json_encode($t); // make it array.
+        }
+    }
+    $options = json_decode($options);
+    
+    return in_array($test, $options, true) ? true : false; // finally!
```

Comment:
Case in point. Who needs their own coding style as a creative outlet when we have comments :)

Reply:
Don't encourage me too much or my code will turn into a book :). I think I may add one more modifier or data type for paragraph. Mainly for print. Static text is fine but getting the text to span entire row is challenge. Will let ya know.